frontend/tests/ui/util.py

```diff
@@ -139,24 +139,25 @@ def assert_no_js_errors(driver, *, path_whitelist=(), message_list=()):
                 continue
             if is_path_whitelisted(entry['message'], path_whitelist):
                 continue
-            if message_matches(entry['message'], message_list):
+            if message_matches(entry['message'], message_whitelist):
                 continue
             unexpected_errors.append(entry['message'])
         assert not unexpected_errors, '\n'.join(unexpected_errors)
 
 
 @contextlib.contextmanager
-def assert_js_errors(driver, *, message=()):
-    '''Shows in a list unexpected errors in javascript console'''
-    expected_error_is_found = True
+def assert_js_errors(driver, *, message_list):
+    '''Asserts that a JavaScript error is logged in the console'''
+    if message_list is None:
```

Comment:
esto debería arrojar una excepción: no es legal pasarle una lista vacía de errores (porque no haría nada):

```suggestion
    assert message_list, 'Message list cannot be empty'
```

Reply:
Listo